tui/common.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+from pyppeteer.page import Page
+
+
+async def accept_cookies(page: Page):
+    button_wrapper = await page.querySelector(".cookies-bar__buttons-wrapper")
+    if button_wrapper is None:
```

Comment:
Możesz tutaj skorzystać z walrus operatora:
```suggestion
    if button_wrapper := await page.querySelector(".cookies-bar__buttons-wrapper"):
        button = await button_wrapper.querySelector("button")
        assert button
        await button.click()
```